app/controllers/dinners_controller.rb

```diff
@@ -0,0 +1,21 @@
+class DinnersController < ApplicationController
+  def new
+    @dinner = Dinner.new
+  end
+
+  def create
+    @dinner = Dinner.new(dinner_params)
+    if @dinner.save
+      # TODO -- Flash a success
+      redirect_to root_path
+    else
+      render 'new'
+    end
+  end
+
+  private
+
+  def dinner_params
+    params.require(:dinner).permit(:dinner_name, :date, :time, :location, :email)
```

Comment:
Don't call the field `dinner_name`.  Imagine how it looks when you'll call it — `dinner.dinner_name` vs. `dinner.name`.